### PR TITLE
Explicitly forbid causing intentional puzzle defects.

### PR DESCRIPTION
**Pull request by lgarron**
Based on an email by Callum Goodyear to the WRC:

https://groups.google.com/d/msg/wca-wrc/Plyqnzwi90M/dkjdxnFyBAAJ

### CODE REVIEW

**Comment by abunickabhi** [comment]
Agree on this change, although analysing the intention is a tough task, if the competitor's cube is on the verge of being defective.

**Comment by xsrvmy** [comment]
Does "correcting a previous defect" include fixing a lockup? (For example: fixing a jammed 7x7 center by taking it out of the puzzle)

**Comment by lgarron** [comment]
> Does "correcting a previous defect" include fixing a lockup? (For example: fixing a jammed 7x7 center by taking it out of the puzzle)

That is covered by 5a, 5b1, and 5b3.

**Comment by lgarron** [comment]
@AlbertoPdRF, could you review? :-D

File: wca-guidelines.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-5><puzzle-defects><puzzledefects> Article 5: Puzzle Defects
 
+- 5b+) [ADDITION] The competitor is not permitted to cause a puzzle defect intentionally, unless correcting a previous puzzle defect (see [Regulation 5b3b](regulations:regulation:5b3b) and [Regulation 5b3c](regulations:regulation:5b3c)). Examples: removing an attached center cap, rotating a corner piece, peeling a sticker.
```
Comment (AlbertoPdRF) [inline wca-guidelines.md]: To be consistent with other Guidelines, I think we should change "Examples:" to something like: "Examples of puzzle defects that can be caused intentionally: removing ...". What do you think?

Comment (lgarron) [reply wca-guidelines.md]: Done! Could you review again?

Comment (AlbertoPdRF) [reply wca-guidelines.md]: LGTM now :)

Comment (lgarron) [reply wca-guidelines.md]: Thanks! 😄

**Comment by Claster** [comment]
Is peeling a sticker also prohibited in Fewest Moves?

**Comment by lgarron** [comment]
> Is peeling a sticker also prohibited in Fewest Moves?

That is a great question. You can already use stickers in FMC to violate quite a few Regulations, though.

**Comment by xsrvmy** [comment]
I think FMC should be excluded. The judge cannot walk around to check for illegal corner twists :P

**Comment by lgarron** [comment]
> I think FMC should be excluded. The judge cannot walk around to check for illegal corner twists :P

There are many things the judge can't practically walk around to check for. 🤷‍♀️

**Comment by xsrvmy** [comment]
I think the better question is, is there any reason to disqualify a competitor from causing a puzzle defect in FMC? I do not see a reason why, say, twisting a corner should have any effect on fmc results, other than being able to solve the puzzle maybe very slightly faster when scrambled.
Also has regulation 5 ever been used in FMC ever?

**Comment by alexmaasswca** [comment]
Really disagree with this applying to FMC, because it has been allowed in the past, so you're causing a time disadvantage for the competitor for no good reason. 

Let's say this becomes a regulation, and Worlds 2019 happens. If you want to enforce this, you have to watch ALL 180 competitors during the WHOLE hour long attempt, for a competitor to twist a corner, which takes less than a second to do. Do you expect that to be feasible to do during the whole entire attempt? And also the two other ones that happen.

In different events you have a judge watching you pretty much the whole solve, which is not the case with FMC, so this is really unenforceable. Why add a reg that that is really unenforceable?

**Pull request merged**